PROCESS.md

```diff
@@ -0,0 +1,40 @@
+# Process
+
+This document describes the practices we use when developing and maintaining `apso`.
```

Comment:
I don't think we need to mention Apso in a monospaced font.

```suggestion
This document describes the practices we use when developing and maintaining Apso.
```

Reply:
Updated! 👍